projects.json

```
@@ -2778,6 +2778,32 @@
       }
     ]
   },
+  {
+    "repository": "Git",
+    "url": "https://github.com/apple/swift-standard-library-preview.git",
+    "path": "swift-standard-library-preview",
+    "branch": "main",
+    "maintainer": "guillaume.lessard@apple.com",
+    "compatibility": [
+      {
+        "version": "5.1",
```

Can we use swift 5.0 here? We would like to only list swift programming lang (4.0, 4.2, and 5.0). 
```suggestion
        "version": "5.0",
```

---

The package manifest has a tools-version of 5.1. It will certainly not build on 5.0 on Linux.

---

the version info is used for swift flag `--swift-version`. Also, we will be building the source compat suite with main and release (5.5) branch.